fabtests/common/shared.c

```diff
@@ -592,6 +592,8 @@ int ft_alloc_msgs(void)
 
 	if (opts.options & FT_OPT_ALLOC_MULT_MR) {
 		ft_set_tx_rx_sizes(&tx_mr_size, &rx_mr_size);
+		tx_mr_size = MAX(FT_MAX_CTRL_MSG, tx_mr_size);
+		rx_mr_size = MAX(FT_MAX_CTRL_MSG, rx_mr_size);
```

Comment:
I think it makes more sense to set these inside the ft_set_tx_rx_sizes function. doesn't make sense to call a function that is supposed to set it and then we modify them. that we can remove the max from lines 605/606 and just multiply it in that line. I think that makes more sense

Reply:
Makes sense, updated